docker/entrypoint-verify-envs.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Dockerfile entrypoint
+set -e
+
+pip install --upgrade pip setuptools
+pip install pipenv
+pipenv --python=3.5
```

Comment:
can we keep this in the Makefile? It allows us to tweak the entire CI pipeline by only editing one file.

Reply:
Yes. I made this file but didn't end up using it anyway. Removing it.